### PR TITLE
Update version handling and add helper script

### PR DESCRIPTION
**Pull request by jjlawren**
## Description

Improve the workflow for updating versions for releases.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated the docstring for new or existing methods
- [x] I have added tests when applicable

### CODE REVIEW

**Comment by jjlawren** [comment]
The `Final` type isn't available in <3.8 🤦

File: plexapi/const.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+"""Constants used by plexapi."""
```
Comment (JonnyWong16) [inline plexapi/const.py:1]: Encoding line?

```
# -*- coding: utf-8 -*-
```

File: version_bump.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    current = Version(const.__version__)
+    bumped = bump_version(current, arguments.bump_type)
+    assert bumped > current, "Bumped version is not newer than old version"
+
+    write_version(bumped)
+
+    if not arguments.commit:
+        return
+
+    subprocess.run(["git", "commit", "-nam", f"Release {bumped}"])
```
Comment (JonnyWong16) [inline version_bump.py:81]: Also create a tag for the version? Add `--tag` argument?

File: version_bump.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
```
Comment (JonnyWong16) [inline version_bump.py:1]: Encoding line?

```
# -*- coding: utf-8 -*-
```

Helper script in root or in the `tools` folder?

**Review by JonnyWong16 — APPROVED** [review]

**Pull request merged**